src/main/java/org/folio/circulation/support/http/server/WebContext.java

```diff
@@ -64,6 +64,10 @@ public String getStringParameter(String name, String defaultValue) {
     return value != null ? value : defaultValue;
   }
 
+  public String getParameter(String name) {
```

Comment:
If this is an overload of `getStringParameter` maybe it should have the same name?

```suggestion
  public String getStringParameter(String name) {
```